build.gradle

```diff
@@ -109,6 +109,7 @@ dependencies {
     runtime "mysql:mysql-connector-java:8.0.23"
     compile "org.codehaus.groovy.modules.http-builder:http-builder:0.7.1"
     compile "org.nibor.autolink:autolink:0.5.0"
+    compile "com.opencsv:opencsv:3.7"
```

Comment:
There's an opencsv 5+ available. However, it uses a different way of building readers.

Reply:
I used opencsv 5.5, however, the syntax was different, I had to revert back to 3.7